peer/brontide.go

```diff
@@ -677,7 +670,65 @@ func (p *Brontide) Start() error {
 
 		return lastSerializedBlockHeader[:]
 	}
-	go p.pingHandler(newPingPayload)
+	// TODO(roasbeef): make dynamic in order to
+	// create fake cover traffic
+	// NOTE(proofofkeags): this was changed to be
+	// dynamic to allow better pong identification,
+	// however, more thought is needed to make this
+	// actually usable as a traffic decoy
+	randPongSize := func() uint16 {
+		return uint16(
+			// We don't need cryptographic randomness here.
+			/* #nosec */
+			rand.Intn(lnwire.MaxPongBytes + 1),
+		)
+	}
+
+	p.pingManager = NewPingManager(&PingManagerConfig{
```

Comment:
can this please be put into the constructor as mentioned before?

Reply:
It was a vestige of the previous iteration where we allocated 3 channels in this method and then used those channels as arguments to the other goroutines. Since we no longer do that, yes, I can move this to the constructor.